src/main/java/evelyn/task/Task.java

```diff
@@ -0,0 +1,48 @@
+package evelyn.task;
+
+/**
+ * Represents a Task.
+ */
+public class Task {
+    private String description;
+    private boolean isMarked;
+
+    /**
+     * Constructor of a Task object
+     * @param desctiption Description of a Task.
+     * @param isMarked Status of a Task.
+     */
+    public Task (String desctiption, boolean isMarked) {
```

Comment:
Perhaps there has been a mistype here and the following line, where you may have entered "desctiption" instead of "description".
```suggestion
    public Task (String description, boolean isMarked) {
```